### ISSUE TITLE
Settings panel size expands on multiple clicks

### ISSUE THREAD

**Issue by ntamblyn** [bitmovin/bitmovin-player-ui#224]
`bitmovin.playerui.version: 3.3.1`

When selecting the UI `buildDemoWithSeparateAudioSubtitlesButtons` and you click on the subtitle list button or audio list button to close the menu the width will increment. 

When clicking on this multiple times the list box will get pretty large.

**Comment by protyposis** [comment]
Thanks for reporting. We will look into it.

**Comment by alexanderte** [comment]
I first thought it was related to subtitles, but it seems like just toggling the settings icon button causes different wrapping/sizing issues. Sometimes the dropdowns are completely hidden. In most (all?) cases it looks correct the first time it is opened.

For this case I’m using 8.2.3 with the default bitmovin-player-ui version.

**Comment by protyposis** [comment]
Yes it gets worse with every time the panel is opened. We have an internal issue open but there's no ETA for a fix at this time.

**Comment by alexanderte** [comment]
Okay, I understand.

Is there a version 8/player UI version combination without this issue?

**Comment by protyposis** [comment]
Unfortunately not. This was introduced with #136 which shipped with UI 3.0.0 (player v8 UI). If this is an issue for you please report it to our support.

**Comment by alexanderte** [comment]
I see. I’ll just stash the current upgrade, and we will stay on version 7 until a fix arrives.
